integration_tests/pages/workAndSkillsPage.ts

```diff
@@ -27,24 +27,22 @@ export default class WorkAndSkillsPage extends Page {
   // CAQ = Courses and Qualifications card
   CAQ_card = (): PageElement => cy.get('#courses-and-qualifications')
 
-  CAQ_header = (): PageElement => cy.get('#courses-and-qualifications > [data-qa="summary-header"] > h2')
+  CAQ_course_table = (): PageElement => cy.get('#completed-in-prison-courses-in-last-12-months-table')
```

Comment:
New cypress methods that use ID or data-qa css selectors; rather than very specific (and fragile) css selectors as before